lib/workers/repository/errors-warnings.ts

```diff
@@ -86,6 +89,52 @@ export function getDepWarningsOnboardingPR(
   return warningText;
 }
 
+export function getPrWarnings(
```

Comment:
If this is new then it should be in a separate PR first

Reply:
it was the getDepPrWarning, i combined it with the encrypted warning

Reply:
separating.